cmd/brew-test-bot.rb

```diff
@@ -688,7 +688,8 @@ def install_bottled_dependent(dependent)
     def formula(formula_name)
       @category = "#{__method__}.#{formula_name}"
 
-      test "brew", "uses", "--recursive", formula_name
+      args = ["--recursive"] unless OS.linux?
+      test "brew", "uses", *args, formula_name
```

Comment:
Any idea why `test-bot` runs `brew uses --recursive formula_name` here and ignores the results, and then runs it a second time below, and saves the results?

Reply:
No clue, will need to read the code more in detail. There are also some new upstream commits that needs to be merged here. I want to fix the build bot here too.
Once this is done, I'll investigate the recursive bug ... if I can find some spare time :)